askama/src/lib.rs

```diff
@@ -510,6 +510,21 @@ pub trait Template {
     }
     /// Renders the template to the given `writer` buffer
     fn render_into(&self, writer: &mut dyn std::fmt::Write) -> Result<()>;
+    /// Helper method which allocates a new `Vec<u8>` and renders into it
+    fn render_bytes(&self) -> Result<Vec<u8>> {
```

Comment:
This doesn't seem to be obviously better than just using `self.render().into()`?

Reply:
My reasoning for introducing that extra method is:
* first of all it is "congruent" with the other functions, thus it is "symmetric";
* secondly, sometimes Rust is quite "magic" at a first glance, and `render().into()` without having the context doesn't say much;  (for example `PathBuf` is one of the possible outcomes of `render().into()`;
* third, if in the future we implement a much more efficient rendering directly into `io::Write`, then such a method would achieve that;
* and finally, in many HTTP servers, the body is a `Vec<u8>`, thus having a `template.render_bytes()` feels more natural;